### DISCUSSION TITLE
Url shortcut for highest release

### DISCUSSION THREAD

**Discussion by stefanak-michal** [community/community#20417] [Other Feature Feedback, Questions, & Ideas]
Hi.

I really like url shortcut for latest release `https://github.com/<repo>/releases/latest` but I have situation when I keep updating older version, which means sometimes latest release is not the highest version. It will be nice if you can introduce new url shortcut `https://github.com/<repo>/releases/highest` which will redirect on highest version available instead of latest.

**Comment by andrea-acampora** [comment]
I think that GitHub uses Semantic Versioning by default so highest and latest refers to the same release.
